design/ingressroute-design.md

```diff
@@ -202,6 +205,11 @@ Contour should allow for an `enforcing` mode which takes in a set of namespaces
 Only those permitted to operate in those namespaces can therefore create virtual hosts and delegate the permission to operate on them to other namespaces. 
 This would most likely be accomplished with a command line flag (`--root-namespaces=[]`) or ConfigMap.
 
+### Disable v1beta1.Ingress
+
+In the scenario where teams want to utilize the `IngressRoute` CRD it may be beneficial to disable Contour from processing `Ingress` resources.
+This can be accomplished by passing the `--disable-k8s-ingress` flag.
+
```

Comment:
I thought we were going to go with RBAC for the initial implementation. That is non admin users do not have the permission to create Ingress objects in any namespace by default, so this can be solved with policy, not code.

Reply:
My notes had a new flag, but RBAC solves just as well. I'll update.